src/main/java/de/sakpaas/backend/model/Occupancy.java

```diff
@@ -7,13 +7,16 @@
 import javax.persistence.Id;
 import javax.persistence.JoinColumn;
 import javax.persistence.ManyToOne;
+import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
 @Getter
 @Setter
 @NoArgsConstructor
+// AllArgsConstructor is needed for the tests
+@AllArgsConstructor
```

Comment:
Meine persönliche Meinung ist, dass wir nie für Tests den eigentlichen Code anpassen sollten. In einzelnen Fällen könnte man die Sichtbarkeit vielleicht ändern (z.B. wenn nötig von private auf proteceted mit @VisibleForTesting)

Hier in dem Fall wird der Konstruktor ja aber nur vom Test benutzt. Mein Vorschlag wäre eine Hilfsmethode im Test die nacheinander die benötigte Setter aufruft. Dann musst du dir trotzdem nur einmal die Arbeit machen, aber unser produktiver Code bleibt saueber

Reply:
Das mit den `Settern` ist eine gute Idee, das hat mich auch echt ange*****zt, dass ich eine neuen Konstruktor erstellen musste. Da hatte ich auch mit @RobSpot-Gaming drüber geredet.

Meine erste Lösung einen Constructor mit `@VisibleForTesting` zu machen ging leider nicht, deswegen musste ich das so machen. Aber dann ändere ich es gerade kurz :)